HyperPlay-Gaming/hyperplay-desktop-client
# [Fix] Web3.Unity Connect

This fixes demon spawn login issue

# To Test
1. Sideload demon spawn survivor browser game `https://www.aintnoking.xyz/dss/`
2. Connect wallet
3. launch game
4. click connect
5. wallet address should show up below menu

---

Use the following Checklist if you have changed something on the Backend or Frontend:
